tests/test_ops/test_deform_conv.py

```diff
@@ -45,9 +45,19 @@ def _test_deformconv(self,
         from mmcv.ops import DeformConv2dPack
         c_in = 1
         c_out = 1
-        x = torch.tensor(input, device=device, dtype=dtype)
+        batch_size = 80
```

Comment:
We can replace `batch_size = 80` with `batch_size = 10` to reduce the memory use when running the unit test.